js-exercises/update-inventory/updateInventory.js

```diff
@@ -0,0 +1,25 @@
+function updateInventory(currentInventory, newInventory) {
+  let updatedInventory = {};
+  let alphabetSort = (array1, array2) => array1[1].charCodeAt(0) - array2[1].charCodeAt(0);
```

Comment:
another option for sorting is **sort()** function

Reply:
This is actually the sorting mechanism @nihal1993 ........I'm using the actual sort function later...